src/h5web/BreadcrumbsBar.tsx

```diff
@@ -48,26 +48,17 @@ function BreadcrumbsBar(props: Props): JSX.Element {
           </span>
         </h1>
       )}
-      <div role="tablist" className={styles.modeToggler}>
-        <button
-          type="button"
-          role="tab"
-          className={styles.btn}
-          aria-selected={!isInspecting}
-          onClick={() => onSetInspecting(false)}
-        >
-          Display
-        </button>
-        <button
-          type="button"
-          role="tab"
-          className={styles.btn}
-          aria-selected={isInspecting}
-          onClick={() => onSetInspecting(true)}
-        >
-          Inspect
-        </button>
-      </div>
+      <ToggleGroup
+        role="tablist"
```

Comment:
The display/inspect toggle group is a `tablist`, not a `radiogroup` like the others. I've decided to make the `role` a mandatory prop instead of choosing a default, as I think it provides valuable information.